testing/sdk_upgrade.py

```diff
@@ -147,6 +147,23 @@ def get_config(package_name, service_name):
     return target_config
 
 
+def update_service(package_name, service_name, additional_options=None, to_package_version=None):
+    update_cmd = ['update', 'start']
+    if to_package_version:
+        update_cmd.append('--package-version={}'.format(to_package_version))
+    if additional_options:
+        with tempfile.NamedTemporaryFile("w") as f:
+            json.dump(additional_options, f)
+            f.flush()  # ensure json content is available for the CLI to read below
+            sdk_cmd.svc_cli(
+                package_name,
+                service_name,
+                '{} --options={}'.format(' '.join(update_cmd), f.name)
+            )
+    else:
+        sdk_cmd.svc_cli(package_name, service_name, ' '.join(update_cmd), check=True)
```

Comment:
As we have helper methods for waiting on deploy, we can ignore to wait so that the callers can explicitly choose to wait. Also, in case of a multi service we do not actually know if we should wait for all the services and what are the names of the services if any. We can address this in a overhaul of testing to support multi service better. Added this to https://jira.mesosphere.com/browse/DCOS-39743